x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -576,13 +594,14 @@ ${JSON.stringify(cypressConfigFile, null, 2)}
                     failedSpecFilePaths.push(filePath);
                   }
                   // Delete serverless project
-                  log.info(`${id} : Deleting project ${PROJECT_NAME}...`);
+                  log.info(`Deleting project ${PROJECT_NAME} and ID ${project.id} ...`);
                   await cloudHandler.deleteSecurityProject(project.id, PROJECT_NAME);
                 } catch (error) {
                   // False positive
                   // eslint-disable-next-line require-atomic-updates
                   result = error;
                   failedSpecFilePaths.push(filePath);
+                  log.error(`Something went wrong with the cypress execution: ${error}`);
```

Comment:
nit: Could we make error message clearer? Something like `Cypress runner failed: ${error}` looks better.